### DISCUSSION TITLE
jbang and proxy settings

### DISCUSSION THREAD

**Discussion by matternikus** [jbangdev/jbang#671] [Q&A] [answered]
I'm looking for a way to make jbang use my proxy settings, i.e. 127.0.0.1:8888 for downloading resources outside of the deps management such as jdk install or simple running. 
My attempts with -Dhttps oder env vars didn't work.

Are there other ways to do it?

Thanks Marcus

**Comment by quintesse** [comment, marked as answer]
Have you tried using the solution mentioned here: https://github.com/jbangdev/jbang/issues/547#issuecomment-734154630 ?
If that doesn't work you could perhaps try this: https://github.com/jbangdev/jbang/issues/447#issuecomment-709591174
We're still waiting on feedback for that one so it would be great if you could share your experiences.

**Reply by matternikus** [reply]
Thanks for the tip.

In my usecase with MobaXterm and a local running cntlm-proxy on 127.0.0.1:8888 a kind of the first one #547 works.

`export JAVA_TOOL_OPTIONS="-Dhttps.proxyHost=127.0.0.1 -Dhttps.proxyPort=8888"`

**Comment by koppor** [comment]
On Windows, following was the solution for me

> can you try make a `c:\Users\{username}\.m2\settings.xml` with something as described in http://maven.apache.org/guides/mini/guide-proxies.html with the settings for your proxy ?

Modified from https://github.com/jbangdev/jbang/issues/447#issuecomment-709591174
